frontend/src/components/smart/SkillBalanceDisplay.vue

```diff
@@ -170,4 +191,29 @@ export default Vue.extend({
 .add-button:hover {
   cursor: pointer;
 }
+.buy-skill-modal {
+  display: flex;
+  justify-content: space-between;
+}
+.buy-skill-modal-child{
+  width: 50%;
+  height: 300px;
+  align-items: center;
+  justify-content: space-between;
+  margin: 10%;
+  display: flex;
+  flex-direction: column;
+}
+.img-apeswap{
```

Comment:
it's the same as .img-transak, so you can just do it like that:
.img-apeswap, 
.img-transak {
  width:100%;
  max-width: 250px;
  height: auto;
  margin-bottom: 30px;
}